malaria-ios/ViewControllers/Info Hub and Games/Games Lists/Games/Rapid Fire/RapidFireViewController.swift

```diff
@@ -73,8 +72,16 @@ class RapidFireViewController: GameViewController {
   }
   
   override func endGame() {
-    super.endGame()
+    // Save the user's final score
+    game.maximumScore = userScore
+    
+    // Send a notification that the game has finished.
+    let dict = ["game": rapidFireGame]
```

Comment:
ok. I undestand what you are doing here. Do you need to pass the dictionary? Can you simplify?